.github/workflows/go.yaml

```diff
@@ -35,3 +35,16 @@ jobs:
           go-version: 1.18
       - name: Test
         run: go test $(go list ./... | grep -v tests) -race -count=1
+  test-windows:
```

Comment:
Why separated, out of interest?

Reply:
We use the race condition checker on linux, and on windows run the usual tests.  Running the race checker on windows is slower than linux, which makes some tests flaky as we sometimes assert latency & timing on tests :(